Altis_Life.Altis/stringtable.xml

```diff
@@ -4658,7 +4658,7 @@
             <Czech></Czech>
             <Spanish>Necesitas %1 para transmitir un anuncio!</Spanish>
             <Russian></Russian>
-            <French>Vous avez besoin de $%1 pour envoyer une diffusion !</French>
+            <French>Vous avez besoin de $%1 pour commencer à diffuser !</French>
```

Comment:
Wrong, because we're not broadcasting like in live or something. It's only one action to send a AAN broadcast.